### PR TITLE
Features/auth

### PR DESCRIPTION
**Pull request by Dercraker**

### CODE REVIEW

File: src/components/passwordReset/RequestResetPasswordModal.tsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        default:
+          console.error(
+            "Impossible de se connecter au serveur d'authentification",
+          );
+          notifications.show({
+            title: "Une erreur s'est produite",
+            message: 'Merci de réessayer plus tard',
+            autoClose: 2000,
+            color: 'red',
+            icon: <IconX />,
```
Comment (Baptiste-Ferrand) [inline src/components/passwordReset/RequestResetPasswordModal.tsx]: 2s is to short for read, as exemple me i can't read  this in 2s

File: src/components/passwordReset/RequestResetPasswordModal.tsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+  const errorNotification = (message: string) =>
+    notifications.show({
+      title: "Une erreur s'est produite",
+      message: message,
+      autoClose: 2000,
+      color: 'red',
+      icon: <IconX />,
+    });
+
```
Comment (Baptiste-Ferrand) [inline src/components/passwordReset/RequestResetPasswordModal.tsx]: 2s si to short for read, and idk the error message, but maybe he can be longer than 24 characteres,
5s better and u can close it manually if you want so..

File: src/pages/confirmMail/ConfirmMailPage.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+          message: "Votre demande n'a pas pu être prise en compte",
+          autoClose: 5000,
+          color: 'red',
+          icon: <IconX />,
+        });
+        navigate('/');
+      });
+  };
+
+  return <></>;
```
Comment (BaptisteLegat) [inline src/pages/confirmMail/ConfirmMailPage.tsx]: On ne return rien ?

File: src/pages/forgotPassword/ForgotPasswordPage.tsx [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+    toggleResetPasswordLoading();
+    try {
+      await AnoAuthController.resetPasswordPOST(
+        resetPasswordForm.values as ResetPasswordDto,
+      );
+      successNotification();
+      navigate('/login');
+    } catch (e: any) {
+      console.log('🚀 ~ handleFromSubmit ~ e:', e.response.data);
```
Comment (BaptisteLegat) [inline src/pages/forgotPassword/ForgotPasswordPage.tsx]: a supprimer

**Review by BaptisteLegat — APPROVED** [review]

**Pull request merged**